.github/workflows/static-analysis.yml

```diff
@@ -59,4 +59,4 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Psalm
-        uses: docker://vimeo/psalm-github-actions:3.17.2
+        uses: docker://vimeo/psalm-github-actions:3.18.0
```

Comment:
Why are you not using the exact same version as in `composer.json`?

Reply:
it is not tagged https://hub.docker.com/r/vimeo/psalm-github-actions/tags

Reply:
Why not upgrade directly to 4.1.1?

Reply:
done